packages/yourloops/lib/team/hook.tsx

```diff
@@ -29,9 +29,10 @@
 import * as React from "react";
 import bows from "bows";
 
+import { MedicalData } from "../../models/shoreline";
 import { ITeam, TeamType, TeamMemberRole, TypeTeamMemberRole, TeamMemberStatus, ITeamMember } from "../../models/team";
 
-import { errorTextFromException } from "../utils";
+import { errorTextFromException, isUserAPatient } from "../utils";
```

Comment:
Utils file start to contains too much different responsibilities. What do you think if we start to split it ? i could start it if you would like.

Reply:
This function is removed in a following PR.
But yes this file has many topics, but for now it's still very small, without really a specific domain.
I think we can wait to split this file.